src/engraving/dom/masterscore.h

```diff
@@ -141,6 +143,11 @@ class MasterScore : public Score
     TempoMap* tempomap() const override { return _tempomap; }
     async::Channel<ScoreChangesRange> changesChannel() const override { return m_changesRangeChannel; }
 
+    BeatsPerSecond tempoPrimo() const override { return _tempoPrimo; }
+    void resetTempoPrimo() { _tempoPrimo = Constants::DEFAULT_TEMPO; _tempoPrimoSet = false; }
+    void setTempoPrimo(BeatsPerSecond tempoPrimo) { _tempoPrimo = tempoPrimo; _tempoPrimoSet = true; }
+    bool tempoPrimoSet() const { return _tempoPrimoSet; }
```

Comment:
It looks like these methods are only used inside `Score::setUpTempoMap()`. Therefore, I think it would be preferable to create a local variable inside `Score::setUpTempoMap()` instead of adding this as a member of the `MasterScore` class.

Reply:
It is used in `Score::setUpTempoMap` and `Score::rebuildTempoAndTimeSigMaps`.
As the latter is also private and only called once from `setUpTempoMap`, I localized the variable in `setUpTempoMap` and pass it as mutable reference paremeter to `rebuildTempoAndTimeSigMaps`.